manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1590,9 +1590,9 @@ spec:
         listener:
           filterChain:
             filter:
-              name: "envoy.http_connection_manager"
+              name: "envoy.filters.network.http_connection_manager"
```

Comment:
Is this going to impact users envoyfilters?

Reply:
Yes. It will